Gaijinizer patch/Map164.txt

```diff
@@ -169,9 +184,9 @@ You've come to here...\\N[1].#MTLed
 まるで止まっていた時が一気に動き出すように……
 > CONTEXT: Map164/events/2/pages/0/13/Dialogue/hyakki
 > CONTEXT: Map164/events/7/pages/0/16/Dialogue/hyakki
-This world has changed dramatically in this short
-time. Just like when you were at a standstill it
-started moving at once ...#MTLed
+This world has changed dramatically in such a
+short time. It's as if stopped time had caught up
+with itself all at once...#RonIffyWording
 > END STRING
```

Comment:
Flow:
>_**This world has changed so much in such
a short period of time. It's as if time was stopped,
and has been quickly trying to catch up...**_